.travis.yml

```diff
@@ -1,50 +1,54 @@
 sudo: false
 branches:
   only:
-    - master
```

Comment:
YAML is weird. Doesn't really matter. I think my editor tries to do this too but I'm not crazy about it. I suppose it will make life easier though so we can leave it.